File: app/src/main/java/com/example/myapplication/atest/HttpErrorCode.java
package com.example.myapplication.atest;

/**
 * @author liuaibin
 */
public interface HttpErrorCode {

    /**
     * for 401
     */
    String UNAUTHORIZED = "Unauthorized";

    /**
     * for 400
     */
    String DEVICE_HAS_ADD = "DeviceHasAdd";
    String DEVICES_NOT_BELONG_THE_USER = "DevicesNotBelongTheUser";
    String EMAIL_HAS_REGIST = "EmailHasRegist";
    String INTERVAL = "Interval";
    String INVALID_PARAM = "InvalidParam";
    String INVALID_TOKEN = "InvalidToken";
    String NOT_SUCH_EMAIL = "NotSuchEmail";
    String NOT_SUCH_USER = "NotSuchUser";
    String VALIDATE_CODE_INVALID = "ValidateCodeInvalid";

    /**
     * for 500
     */
    String INTERNAL_SERVER_ERROR = "InternalServerError";
}

File: app/src/main/java/com/example/myapplication/atestkotlin/HttpErrorCode.kt
package com.example.myapplication.atestkotlin

/**
 * Copyright (c) 2022 Raysharp.cn. All rights reserved.
 *
 * HttpErrorCode
 * @author liujianming
 * @date 2022-01-05
 */
class HttpErrorCode {

    /**
     * for 401
     */
    var UNAUTHORIZED = "Unauthorized"

    /**
     * for 400
     */
    var DEVICE_HAS_ADD = "DeviceHasAdd"
    var DEVICES_NOT_BELONG_THE_USER = "DevicesNotBelongTheUser"
    var EMAIL_HAS_REGIST = "EmailHasRegist"
    var INTERVAL = "Interval"
    var INVALID_PARAM = "InvalidParam"
    var INVALID_TOKEN = "InvalidToken"
    var NOT_SUCH_EMAIL = "NotSuchEmail"
    var NOT_SUCH_USER = "NotSuchUser"
    var VALIDATE_CODE_INVALID = "ValidateCodeInvalid"

    /**
     * for 500
     */
    var INTERNAL_SERVER_ERROR = "InternalServerError"
}